kubernetes/docs/deploy-to-ibmcloud.md

```diff
@@ -0,0 +1,165 @@
+# Deploy to IBM Cloud
+
+## Create a Kubernetes cluster
+
+https://cloud.ibm.com/kubernetes/clusters
+
+![clusters](../imgs/clusters.png)
+
+Click on `shopping-cluster` to view:
+
+![shopping-cluster](../imgs/shopping-cluster.png)
+
+## Set up CLI for IBM Cloud
+
+https://cloud.ibm.com/docs/containers?topic=containers-cs_cli_install
+
+## Connect to IBM Cloud with CLI
+
+If this is your first time connecting to an IBM Cloud cluster, see the
+[full setup directions](https://cloud.ibm.com/docs/containers?topic=containers-cs_cli_install).
+
+![connect-via-cli](../imgs/connect-via-cli.png)
+
+## Set up helm
+
+https://cloud.ibm.com/docs/containers?topic=containers-helm
+
+```sh
+kubectl create serviceaccount tiller -n kube-system
+kubectl create clusterrolebinding tiller --clusterrole=cluster-admin --serviceaccount=kube-system:tiller -n kube-system
+kubectl get serviceaccount -n kube-system tiller
+helm init --service-account tiller
+```
+
+## Set up IBM Cloud container registry
+
+https://cloud.ibm.com/kubernetes/registry/main/start
+
+## Upload images to IBM Cloud container registry
+
+1. Login into IBM Cloud container registry
+
+```sh
+ibmcloud cr login
+```
+
+2. Use an existing namespace for the container registry or create a new one:
+
+Please substitute `<your-namespace>` to container registry namespace you create
+for IBM Cloud.
+
+Existing namespaces can be listed:
+
+```sh
+ibmcloud cr namespaces
+```
+
+You can add a namespace as follows:
+
+```sh
+ibmcloud cr namespace-add <your-namespace>
+```
+
+3. Upload docker images
```

Comment:
There's no mention of running `docker:build` . We should have them run it if they haven't done so yet.